elements/rh-tabs/rh-tab-panel.css

```diff
@@ -0,0 +1,21 @@
+#rhds-container {
+  display: block;
+  padding: var(--rh-space-2xl, 32px);
+}
+
+:host(:not([vertical]):not([box]):not([inset])) #rhds-container {
+  padding-inline-start: 0;
```

Comment:
The panel tabs with this variant should be using padding: var(--rh-space-2xl, 32px) too so that the first tab's text lines up with the text in the panel.